### PR TITLE
Add ignore-2.12.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Please, after merge, add 2.11 to the test matrix (I didn't find it)

https://github.com/ansible-collections/overview/issues/45#issuecomment-813821100

### CODE REVIEW

**Comment by NilashishC** [comment]
@Andersson007 Thank you for this PR! A trivial changelog fragment is needed for CI to pass. Could you please add that too?

**Comment by Andersson007** [comment]
@NilashishC done, thanks

File: changelogs/fragments/125-add_ignore_txt.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+trivial:
```
Comment (Andersson007) [inline changelogs/fragments/125-add_ignore_txt.yml:1]:
```suggestion
---
trivial:
```

File: changelogs/fragments/125-add_ignore_txt.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+- Add ingore-2.12.txt.
```
Comment (Andersson007) [inline changelogs/fragments/125-add_ignore_txt.yml:3]:
```suggestion
  - Add ingore-2.12.txt.
```

**Comment by Andersson007** [comment]
ready for review

**Review by ashwini-mhatre — APPROVED** [review]

**Pull request merged**